.travis.yml

```diff
@@ -1,51 +1,9 @@
-language: php
+# https://docs.travis-ci.com/user/build-config-imports/
```

Comment:
probably don't need this comment in the file

Reply:
done